npm/janus.d.ts

```diff
@@ -241,7 +343,7 @@ declare namespace JanusJS {
 		getServer(): string;
 		isConnected(): boolean;
 		reconnect(callbacks: ReconnectOptions): void;
-		getSessionId(): number;
+		getSessionId(): string;
```

Comment:
`number`

Reply:
I am really sorry that I have so many negligence, then I will address these points in detail next, thanks for the review